awx/api/conf.py

```diff
@@ -1,30 +1,48 @@
 # Django
 from django.utils.translation import ugettext_lazy as _
 
-# Tower
+# AWX
 from awx.conf import fields, register
+from awx.api.fields import OAuth2ProviderField
 
 
+# register(
+#     'AUTH_TOKEN_EXPIRATION',
+#     field_class=fields.IntegerField,
+#     min_value=60,
+#     label=_('Idle Time Force Log Out'),
+#     help_text=_('Number of seconds that a user is inactive before they will need to login again.'),
+#     category=_('Authentication'),
+#     category_slug='authentication',
+# )
+# 
+# register(
+#     'AUTH_TOKEN_PER_USER',
+#     field_class=fields.IntegerField,
+#     min_value=-1,
+#     label=_('Maximum number of simultaneous logins'),
+#     help_text=_('Maximum number of simultaneous logins a user may have. To disable enter -1.'),
+#     category=_('Authentication'),
+#     category_slug='authentication',
+# )
```

Comment:
What's going on with the commented settings? Looks like we can get rid of these lines?